### ISSUE TITLE
Handling of operational attributes appears broken/changed between 2.4.0 and 2.4.1

### ISSUE THREAD

**Issue by daenney** [cannatag/ldap3#494]
On ldap3 2.4.0:

```python
def search(conn, dn, obj):
    """Search a DN with scope SUBTREE for objects matching the filter."""
    obj = ObjectDef(obj, conn)
    r = Reader(conn, obj, dn, get_operational_attributes=True)
    r.search()
    return r


ds = Conn(...)
users = 'ou=people,dc=....'
USER_CLASSES = ['inetOrgPerson', ...]
search(ds, users, USER_CLASSES)
```

This works fine.

However, in 2.4.1, the same code results in:
```python
ldap3.core.exceptions.LDAPCursorError: attribute 'memberOf' not in object class 'top, person, organizationalPerson, inetOrgPerson, ...' for entry uid=alice,ou=people,...
```

I'm not sure why though.

**Comment by maxsond** [comment]
Could this be caused by https://github.com/cannatag/ldap3/issues/493? That would imply that the dev branch already has this fixed. I checked and it looks like 2.4.0 still uses LDAPKeyError in EntryBase.__getitem__, whereas 2.4.1 uses LDAPCursorError.

**Comment by cannatag** [comment]
Probably. Can you test the code in dev to check if it is working now?

**Comment by cannatag** [comment]
Fixed in v2.5.
